src/WebSockets.jl

```diff
@@ -71,17 +71,17 @@ function send_fragment(ws::WebSocket, islast::Bool, data::Array{Uint8}, opcode=0
   b1::Uint8 = (islast ? 0b1000_0000 : 0b0000_0000) | opcode
   if l <= 125
     write(ws.socket,b1)
-    write(ws.socket,uint8(l))
+    write(ws.socket,convert(Uint8,l))
```

Comment:
I hadn't heard about Uint8 and such going away. Can you point me to the
discussion? What's replacing it?

On Thu, Jul 2, 2015 at 5:20 PM, Sean Garborg <notifications@github.com>
wrote:

> In src/WebSockets.jl
> <https://github.com/JuliaWeb/WebSockets.jl/pull/24#discussion_r33788719>:
>
> > @@ -71,17 +71,17 @@ function send_fragment(ws::WebSocket, islast::Bool, data::Array{Uint8}, opcode=0
> >    b1::Uint8 = (islast ? 0b1000_0000 : 0b0000_0000) | opcode
> >    if l <= 125
> >      write(ws.socket,b1)
> > -    write(ws.socket,uint8(l))
> > +    write(ws.socket,convert(Uint8,l))
>
> Since Uint8 and friends are going away, spelling-wise, my preference is
> to bite the bullet and add Compat.jl as a dependency (@compat(UInt8(l))).
> Most packages benefit from using Compat in more than one place/way, so
> 'bite the bullet' may be a little dramatic. This certainly works as is,
> though, if you want to use convert for now.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/JuliaWeb/WebSockets.jl/pull/24/files#r33788719>.
>